erizo_controller/installErizo_controller.sh

```diff
@@ -21,13 +21,13 @@ check_result() {
 echo [erizo_controller] Installing node_modules for erizo_controller
 
 nvm use
-npm install --loglevel error amqp socket.io@2.0.3 log4js@1.0.1 node-getopt uuid@3.1.0 sdp-transform@2.3.0
+npm install --loglevel error amqp socket.io@2.0.3 log4js@1.0.1 node-getopt uuid@3.1.0 sdp-transform@2.3.0 socket.io-client@2.1.0 webrtc-adapter@6.1.5 webpack-conditional-loader@1.0.12
```

Comment:
`socket.io-client@2.1.0` -> consider using `socket.io-client@2.0.3` which is the one we're using in spine.

Reply:
ok np

Reply:
Why don't we also update spine socket.io version instead? Version 2.1.1 introduced only bug fixes with respect the 2.0.3, isn't it better to keep things updated?

Reply:
Yes, but in it's own PR, please.